### ISSUE TITLE
Simple recipe resizer

### ISSUE THREAD

**Issue by xboxhaxorz** [TandoorRecipes/recipes#3317]
### Describe the solution you'd like

Sometimes i come across a recipe and i dont know yet if i want to put it in tandoor, i want to make the recipe to test but i might want to modify the serving size, so i use this site https://bakingcalculators.com/resizerecipe/Default.aspx

Perhaps that is something that could be implemented and can show in the menu next to the books link

**Comment by vabene1111** [comment]
yes, I would really like something like this in the edit view 👍 leaving this open for the future :)

**Comment by T7alabdullah** [comment]
Hello! I’m a student at Carnegie Mellon University in Qatar, and my team and I are working on a Software Engineering project for our course. We’re required to contribute to open-source projects, and this issue caught our attention—it looks like a great fit for what we’re learning right now.

We understand that no features like this will be merged until the migration to #2976 is complete, but we’d still like to tackle this issue in the meantime as part of our project. It would be a great learning opportunity for us, and we’ll make sure to align with the repository’s guidelines and structure.

Let us know if that works for you!

**Comment by vabene1111** [comment]
Hi, feel free to give it a try. If its well enough implemented I might merge it in the old version, if not i might still be able to use it as a basic concept for how to implement this for the new version
